packages/core-typings/src/cloud/CloudSyncAnnouncement.ts

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
Is this really necessary? The file doesn't seem to be infringing any naming-convention rules. Can you check?